CONTRIBUTING.md

```diff
@@ -0,0 +1,228 @@
+    shared manually, instead print instructions for how to access the dataset,
+    and where to put it when it is downloaded.
+
+    Parameters
+    ----------
+    data_home: str or None, default=None
+        Download the dataset to a subfolder of this folder.
+        If None, uses the default location specified by MIR_DATASETS_DIR.
+    clobber: bool, default=False
+        If True, overwrites an existing copy of the dataset with the downloaded
+        version.
+        If False, if the dataset already exists, does not overwrite.
+    """
+    pass  # see orchset.py for a specific example
+
+
+def validate(data_home):
+    """Check that all of the files referenced in the index exist on disk.
+
+    Returns
+    -------
+    missing_files : list
+        List of expected absolute paths of missing files.
+    invalid_checksums : list
+        List of absolute paths where checksums do not match.
+    """
+    # keys in EXAMPLE_INDEX which map to filepaths
+    missing_files, invalid_checksums = validator(
+        EXAMPLE_INDEX, data_home)
+    return missing_files, invalid_checksums
+
+
+def track_ids():
+    """Get a list of the dataset's track IDs.
+
+    Returns
+    -------
+    List of the dataset's track IDs
+    """
+    return list(EXAMPLE_INDEX.keys())
+
+
+def load(data_home=None):
+    """Load the dataset to a dictionary of track objects.
+
+    Parameters
+    ----------
+    data_home: str or None, default=None
+        Path where the dataset is stored.
+        If None, uses the default MIR_DATASETS_DIR.
+
+    Returns
+    -------
+    data_dict: dict
+        Dictionary keyed by track ID with instances of ExampleTrack as values.
+    """
+    validate(data_home)
+    data_dict = {}
+    for key in track_ids():
+        data_dict[key] = load_track(key, data_home=data_home)
+    return data_dict
+
+
+def load_track(track_id, data_home=None):
+    """Load a track for a given track ID.
+
+    Parameters
+    ----------
+    track_id: str
+        Unique identifier for the track.
+    data_home: str or None, default=None
+        Path where the dataset is stored.
+        If None, uses the default MIR_DATASETS_DIR.
+
+    Returns
+    -------
+    example_track: ExampleTrack
+        An instance of ExampleTrack with the relevant data filled in.
+    """
+    if track_id not in EXAMPLE_INDEX.keys():
+        raise ValueError(
+            "{} is not a valid track ID in Example".format(track_id))
+    track_data = EXAMPLE_INDEX[track_id]
+
+    return ExampleTrack(
+        track_id,
+        track_data['audio_path'],
+        track_data['annotation_path'],
+        track_data['genre']
+    )
+
+
+def cite():
+    """Print citation data in two formats.
+    """
+    cite_data = """
+===========  MLA ===========
+TODO: The MLA format citation here
+
+========== Bibtex ==========
+TODO: The Bibtex format citation here
+"""
+
+    print(cite_data)
+```
+
+
+## Adding tests to your dataset
+
+Add some words about adding tests here!
```

Comment:
@tkell take a look at my tests for `ikala`. I think most people will be a bit overwhelmed by the pytest boilerplate, the mocking, etc. I think we should advise people to

1) make a fake/tiny version of the dataset in the tests folder `tests/resources/mir_datasets/my_dataset/`
2) test all of the dataset specific code, e.g. the Track object, any of the _load functions, etc, as I did in ikala.
3) let's give people permission to skip writing tests for the `download` and `validate` functions...maybe we can find a clever way to test all loader's `download` and `validate` functions in a separate test file

Reply:
Took a shot at the copy for this - please correct the prose as needed.